pkgs/top-level/all-packages.nix

```diff
@@ -20753,11 +20753,7 @@ in
 
   calculix = callPackage ../applications/science/math/calculix {};
 
-  calibre-py2 = libsForQt5.callPackage ../applications/misc/calibre { pythonPackages = python2Packages; };
```

Comment:
Do we want to add an alias for this to notify people that it got removed?

Reply:
Didn't thought of that sorry, could be great yes!

Reply:
Is that what you're thinking about? 
```nix
  calibre = libsForQt5.callPackage ../applications/misc/calibre {};

  calibre-py2 = calibre;

  calibre-py3 = calibre;
```